.github/workflows/router-ci.yaml

```diff
@@ -184,12 +184,16 @@ jobs:
           image_name: router
           image_description: "Cosmo Router"
           image_platforms: 'linux/amd64'
+          load_Image: 'true'
           push: 'false'
 
+      - run: podman images
+
```

Comment:
Looks like a debug instruction. Can this be removed?

Reply:
Good catch